src/QueryFilterHelpers.ts

```diff
@@ -634,3 +654,128 @@ export function executeIntervalELM(
     return null;
   }
 }
+
+/* Interface to aid in parsing when we know "CalendarAgeInYearsAt" is the referenced function */
+interface CalendarAgeInYearsAtRef extends ELMFunctionRef {
+  name: 'CalendarAgeInYearsAt';
+  libraryName: 'Global';
+  operand: [CalendarAgeInYearsDateTime, ELMFunctionRef | ELMProperty | ELMStart | ELMEnd];
+}
+
+/* Interface to aid in parsing when we know "CalendarAgeInYearsAt" is the referenced function. This is for the first operand. */
+interface CalendarAgeInYearsDateTime extends ELMToDateTime {
+  operand: ELMFunctionRef;
+}
```

Comment:
My thought was to not put it in ELMTypes because they would only be used here. This question does have my brain juices flowing and has me wondering if I should split up this file to move the cases for specific expressions into their own files because of how hefty most of them are.

Reply:
I guess these are specific cases that we're handling rather than base ELM types, so that makes sense. Would it make sense to have the types for these custom cases as their own file or nah? Fine either way

Reply:
I think embedding them in the same file is appropriate. It helps to keep the context and makes sure they aren't misused elsewhere.